README.md

```diff
@@ -93,32 +50,54 @@ In order to allow the assistant user to create location reservations and service
     - IBM TRIRIGA Request Central, or
     - IBM TRIRIGA Workplace Reservation Manager
 
-2.	Make note of the user name and password because it will need to be provided to IBM.
+2.	Make note of the user name and password because it will needed in the provisioning step below.
 3.	Important: Do NOT give the assistant user a primary location.  This user will be used to book rooms and submit service requests on behalf of all users in the TRIRIGA instance.
 
-#### E) (OPTIONAL) TEST THE OSLC ENDPOINTS.
+
+#### C) ADD THE CERTIFICATE AUTHORITY CERTIFICATES TO WEBSERVER JAVA RUNTIME ENVIRONMENT
+
+The provisioning process requires a POST call to the IBM TRIRIGA Assistant services through an Integration Object in TRIRIGA.  In order for this call to be secure, the CA certificates need to be added the JRE used by the WebServer running TRIRIGA.  These instructions are for WebSphere.
+
+1. Make a backup of the `cacerts` file located at `WEBSPHERE_ROOT/AppServer/java/8.0/jre/lib/security`.   
+2. Copy the CA certificates file `triassistant_cacerts` from the `cacerts` directory from this GitHub repository to that same directory.
+3. Execute the `keytool` command to add the CA certificates in `triassistant_cacerts` to the existing `cacerts` file:
+    `../../bin/keytool -import -trustcacerts -alias triassistant -file triassitant-certs -keystore ./cacerts -storepass changeit` 
+4. Restart TRIRIGA
+
+#### D) (OPTIONAL) TEST THE OSLC ENDPOINTS.
```

Comment:
Is there a simple specific curl we could toss in rather than point them at the whole postman collection?

Reply:
so I wrote up these instructions...

Execute a quick `curl` command to be sure that provisioning will be successful.  Follow these steps to make sure you have the necessary info and objects in place.

1.  You'll need the base64 encoded representation of the usernmame and password for the assistant account created earlier.  The `base64` unix command will generate this string for you given the username and password.
   `echo "username:password" | base64`
2.  Execute the following `curl` command and replace TRIRIGA_URL and ENCODED_USERNAME_AND_PASSWORD with your info.
    `curl --location --request GET 'TRIRIGA_URL/oslc/spq/ibmWAProvisionQC?oslc.select=*' --header 'Authorization: Basic ENCODED_USERNAME_AND_PASSWORD`

and when I tried them I got this error:

curl: (60) SSL: no alternative certificate subject name matches target host name 'triassistant.stage.maximo.com'
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.

I don't get this error with Postman.

Reply:
adding a -k or --insecure should remedy that I would think

Reply:
Don't you have to change a setting in postman to fix the same issue, I thought I recalled needing to change something to get the collection to work.

Reply:
k, I'll leave the --insecure for now.